.github/workflows/lint.yml

```diff
@@ -3,8 +3,8 @@ name: Lint Files
 on:
   push:
     branches: [ main, main-nuxt ]
-  pull_request:
-    branches: [ main, main-nuxt ]
+  # pull_request:
+  #   branches: [ main, main-nuxt ]
```

Comment:
why not lint on PRs ? 👀

Reply:
At the moment reviewdog is configured to look at all files and run eslint so it would be like doing the same action twice